### PR TITLE
Added the ability to suppress echo when entering sensitive parameters…

### PR DESCRIPTION
**Pull request by SandyChapman**
… such as passwords.

### CODE REVIEW

**Comment by SandyChapman** [comment]
Let me know if this isn't a suitable addition or if you'd like a different name than `sensitive`.

**Comment by SandyChapman** [comment]
@KrauseFx : Any idea if this is an acceptable PR? What are the next steps to get this merged?

File: fastlane_core/lib/fastlane_core/configuration/configuration.rb [hunk 1]
```diff
@@ -200,7 +200,7 @@ def fetch(key, ask: true)
 
       while value.nil?
         UI.important("To not be asked about this value, you can specify it using '#{option.key}'")
-        value = ask("#{option.description}: ")
+        value = ask("#{option.description}: ") { |q| q.echo = !option.sensitive }
```
Comment (KrauseFx) [inline fastlane_core/lib/fastlane_core/configuration/configuration.rb]: Shouldn't this be `q.echo = "*" unless option.sensitive`

**Review by KrauseFx — COMMENTED** [review]
Thanks for the pull request, this looks like a nice addition to _fastlane_. We could automatically hide those values in `print_table.rb` too, right? 
In which areas in _fastlane_ do you plan on start using this new property?

File: fastlane_core/lib/fastlane_core/configuration/configuration.rb (hunk 1 above)
Comment (mfurtak) [inline fastlane_core/lib/fastlane_core/configuration/configuration.rb]: I think the code is OK as written if we want the result to be that sensitive things don't have any echoed input.

I think your suggestion would make all non-sensitive params show '*' as their echoed input.

`q.echo = '*' if option.sensitive` would also be good, if we want sensitive things to show asterisks as the echoed input.

https://github.com/JEG2/highline#usage

**Comment by mfurtak** [comment]
Ping to all on this - do we want to resolve conflicts and carry this forward?

**Comment by mfurtak** [comment]
Being carried forward in #7169

**Pull request closed without merging**